tests/integration/targets/dcnm_res_manager/tests/dcnm/dcnm_res_manager_invalid_params.yaml

```diff
@@ -0,0 +1,130 @@
+##############################################
+##               SETUP                      ##
+##############################################
+
+- name: Remove local log file
+  local_action: command rm -f res_mgr.log
+
+- block:
+
+##############################################
+##                MERGE                     ##
+##############################################
+
+    - name: Create Resources - Invalid Pool type
+      cisco.dcnm.dcnm_res_manager:
+        state: merged                                # choose form [merged, deleted, query]
+        fabric: "{{ ansible_it_fabric }}" 
+        config:
+          - entity_name: "l3_vni_fabric"             # A unique name to identify the resource
+            pool_type: "IDLE"                          # choose from ['ID', 'IP', 'SUBNET']
+            pool_name: "L3_VNI"                  # Based on the 'poolType', select appropriate name
+            scope_type: "fabric"                     # choose from ['fabric', 'device', device_interface', 'device_pair', 'link']             
+            resource: "101"                          # The value of the resource being created
+      ignore_errors: yes
```

Comment:
I see you are using the `ignore_errors` flag since these are invalid cases but should we also be storing the result and checking the error message or at least that an error was generated?

Reply:
Will add the checks